head/head.go

```diff
@@ -132,23 +132,40 @@ func NewHead(ctx context.Context, options ...opts.Option) (*Head, chan Bootstrap
 	go func() {
 		// ❓ what is this limiter for?
 		if cfg.Limiter != nil {
-			cfg.Limiter <- struct{}{}
+			select {
+			case cfg.Limiter <- struct{}{}:
```

Comment:
Do we know what the limiter is for now?

Reply:
As far as I can tell, it's a mechanism to stagger bootstrap requests so it doesn't appear like we have 200+ IPFS nodes attacking the bootstrappers.

Reply:
https://github.com/ipfs/dht-node/commit/fee98576a6662041003ff64d6196a1c52ced1315 this is all I got, is there anything else to know @Kubuxu?

Reply:
Yup, it is just to limit how many bootstraps happen concurrently. Probably to not overload network stack, I can't recall.